### PR TITLE
post-rgsoc internships

### PR DESCRIPTION
**Pull request by berlintam**
as discussed in the organization repo. 

@anikalindtner @FloorD @carpodaster @alicetragedy @sareg0

### CODE REVIEW

**Comment by guilleiguaran** [comment]
Maybe a good idea ask also for companies looking for remote candidates :smiley:

**Comment by robertomiranda** [comment]
@guilleiguaran :+1:

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+##### Macedonia
+* Skopje
+
+##### India
+* Pune
+* Mysore
+
+##### Colombia
+* Barranquilla,
```
Comment (carpodaster) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: There is a trailing comma

**Comment by carpodaster** [comment]
Nice post, Tam!

Maybe sort the locations alphabetically?

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Why not offer an internship after the summer is over?
+layout: post
+created_at: Mon Aug 11 2014
```
Comment (carpodaster) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: And yeah, better update this for today + `git mv` the file to reflect the new publish date.

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+##### USA
+
+* Atlanta, GA
+* Washington, D.C.
+* Seattle, WA
+* Chicago, IL
+* Los Angeles, CA
+
+If you’re looking for talented women who are eager to continue learning, or just want more info about how you can help out after RGSoC, please [get in touch](summer-of-code@railsgirls.com)!
```
Comment (carpodaster) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: The email address needs a `mailto:` prefix

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+title: Why not offer an internship after the summer is over?
+layout: post
+created_at: Mon Aug 11 2014
+permalink: blog/rgsoc-internships
+current: blog
+author: Tam
+twitter: travelingtamm
+---
+
+The summer is reaching its halfway mark, so we're hard at work thinking about what to do when it ends. For a handful of selected women, RGSoC has marked a turn in their lives: they have spent hours coding, learning, and discovering a new world of programming possibilities.
```
Comment (svenfuchs) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: > The summer is reaching its halfway mark, so we're hard at work thinking about what to do when it ends. 

That sounds a little like we don't know what we're doing here :) Maybe just omit the second part of the sentence?

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+But sooner than they (and we!) realize, their summer will be over, and it will be time to move on. But to where?
```
Comment (svenfuchs) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: > But to where?

Same here. I'd just omit this one ...

File: blog/_posts/2014-08-11-post-rgsoc-internships.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+That’s where you come in. We are looking for awesome companies _all around the world_ who are willing to offer internships or junior developer positions to Rails Girls Summer of Code alumni so that they can keep moving down this exciting path and honing their skills.
```
Comment (svenfuchs) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: > That’s where you come in.

... and this.

**Comment by svenfuchs** [comment]
Awesome post, @berlintam! I've added a few stylistic comments inline.

File: blog/_posts/2014-08-11-post-rgsoc-internships.md (hunk 6 above)
Comment (carpodaster) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: Personally, I'd leave it in. There's no shame in the fact that RGSoC is not an autonomous body that can heave all the work on its own.

Some degree of _»we need you«_ is definitely in order.

Uncle Sam

Comment (svenfuchs) [inline blog/_posts/2014-08-11-post-rgsoc-internships.md]: I've learned to reduce, reduce, reduce text, in blog posts and other writing, and remove anything that isn't actually required for the one main core message. The message "we need you" is still sent, even if no words are used to make it explicit. The core message is "Tell us about your internship or junior positions"

**Comment by berlintam** [comment]
Thanks @svenfuchs and @carpodaster ! I've made some edits based on your suggestions. I changed the date to tomorrow, so I'll bug @anikalindtner about it then!

@guilleiguaran & @robertomiranda thanks for your suggestions, of course it's up to the individual students if they work remotely or not. I suppose this is something that can be worked out between companies and students if the opportunity presents itself.

**Pull request merged**